### PR TITLE
Add information to screenshots page

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR adds an informational disclaimer to the screenshots page, stating that it shows the screenshots prior to function reduction on May 1.

## What was changed

- https://github.com/corona-warn-app/cwa-website/blob/master/src/data/screenshots.json
- https://github.com/corona-warn-app/cwa-website/blob/master/src/data/screenshots_de.json

## Screenshots
| English | German |
|---|---|
|  |  |

---

**Fixes https://github.com/corona-warn-app/cwa-website/issues/3481**

---
Internal Tracking ID: [EXPOSUREAPP-14634](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-14634)

### CODE REVIEW

File: src/data/screenshots.json [hunk 1]
```diff
@@ -10,7 +10,7 @@
         "content": {
             "textblock": [
                 "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since 19. April, 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
-                "With this update we are fixing bugs in the app. It does not contain any new features.",
+                "With this update we are fixing bugs in the app. It does not contain any new features. The screenshots page displays the implementation prior to the function reduction on May 1, 2023. For more information, please refer to the <a href='../blog/2023-05-02-reduction-of-functions/' target='_blank'>blog</a>.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:13]: Please fix the date in the previous line (not your error!).

It should say "April 19, 2023" in English format.

Comment (Ein-Tim) [reply src/data/screenshots.json:13]: Resolved with https://github.com/corona-warn-app/cwa-website/commit/f56fa649bc7f1bf9a4605566477be705703b95e3

File: src/data/screenshots.json [hunk 2]
```diff
@@ -9,8 +9,8 @@
         },
         "content": {
             "textblock": [
-                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since 19. April, 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
-                "With this update we are fixing bugs in the app. It does not contain any new features.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 3.2 (available since April, 19 2023). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
```
Comment (MikeMcC399) [inline src/data/screenshots.json:12]: "April, 19 2023" > "April 19, 2023"

the comma is in the wrong place

Comment (Ein-Tim) [reply src/data/screenshots.json]: Sorry, I fixed that. I shouldn't do edits on my phone :D

Comment (MikeMcC399) [reply src/data/screenshots.json]: @Ein-Tim 

> Sorry, I fixed that. I shouldn't do edits on my phone :D

It's usually better to have a bigger screen and this is not an urgent change, so wait until you have the right equipment available.

**Review by MikeMcC399 — APPROVED** [review]

**Comment by MikeMcC399** [comment]
@Ein-Tim 

> The failing check is not my fault, right?

- No. It's not a problem with your PR. See https://github.com/corona-warn-app/cwa-website/issues/3479

**Review by dsarkar — APPROVED** [review]

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 Thanks for the PR!

**Pull request merged**